src/adapter/completions.ts

```diff
@@ -143,18 +143,28 @@ export class Completions {
     const source = parseProgram(options.expression);
     const offset = new PositionToOffset(options.expression).convert(options.position);
     let candidate: () => Promise<ICompletionWithSort[]> = () => Promise.resolve([]);
+    let parent: Node | undefined = undefined;
 
     traverse(source, {
       enter: node => {
```

Comment:
estraverse actually gives the parent as the second argument to this function call